cmd/edit.go

```diff
@@ -0,0 +1,138 @@
+// Copyright © 2017 The Kubicorn Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/kris-nova/kubicorn/cutil/logger"
+	"github.com/kris-nova/kubicorn/state"
+	"github.com/kris-nova/kubicorn/state/fs"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"os"
+	"os/exec"
+)
+
+type EditOptions struct {
+	Options
+	Editor string
+}
+
+var eo = &EditOptions{}
+
+var editCmd = &cobra.Command{
+	Use:   "edit <NAME>",
+	Short: "Edit a cluster state",
+	Long:  `Use this command to edit a state.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) > 1 {
```

Comment:
In case len of args is zero, (I think) we should try to read from env variable `KUBICORN_NAME`.

Such as in this case https://github.com/kris-nova/kubicorn/blob/master/cmd/apply.go#L47